### PR TITLE
fix: already open Yeoman UI generator is closed on opening different …

### PR DESCRIPTION
**Pull request by alex-gilin**
…Yeoman UI generator

### CODE REVIEW

File: packages/backend/src/extCommands.ts [hunk 1]
```diff
@@ -29,32 +29,64 @@ export class ExtCommands {
   }
 
   private async yeomanUIPanel_runGenerator_Command() {
-    return (await this.getYeomanUIPanel()).runGenerator();
+    try {
+      return (await this.getYeomanUIPanel()).runGenerator();
+    } catch (e) {
+      console.log(e);
```
Comment (veredcon) [inline packages/backend/src/extCommands.ts:35]: there is no meaning of console.log in the backend, there is already a logger you can use if you want

File: packages/backend/src/extCommands.ts [hunk 2]
```diff
@@ -29,32 +29,64 @@ export class ExtCommands {
...
+    }
   }
 
   private async yeomanUIPanel_loadYeomanUI_Command(uiOptions?: any) {
-    return (await this.getYeomanUIPanel()).loadWebviewPanel(uiOptions);
+    try {
+      return (await this.getYeomanUIPanel()).loadWebviewPanel(uiOptions);
+    } catch (e) {
+      console.log(e);
```
Comment (veredcon) [inline packages/backend/src/extCommands.ts:43]: same as above - logger

File: packages/backend/src/extCommands.ts [hunk 3]
```diff
@@ -29,32 +29,64 @@ export class ExtCommands {
...
+    }
+  }
+
+  private async isInEmptyState(): Promise<boolean> {
+    if (this.yeomanUIPanel?.yeomanui?.generatorName) {
+      if (
+        (await window.showWarningMessage(
+          `The generator ${
+            this.yeomanUIPanel.yeomanui.generatorName.split(":")[0]
+          } is not completed yet. Are you sure you want to stop it before it finished?`,
```
Comment (veredcon) [inline packages/backend/src/extCommands.ts:69]: Reviewed by Paola?

**Review by veredcon — APPROVED** [review]

File: packages/backend/src/extCommands.ts (hunk 3 above)
Comment (alex-gilin) [reply packages/backend/src/extCommands.ts]: changed

**Pull request merged**